app/assets/stylesheets/views/index.scss

```diff
@@ -0,0 +1,12 @@
+.results-table-wrapper {
+  .govuk-table__body {
```

Comment:
I would move these to `app/assets/stylesheets/utilities/_overwrites.scss` as it's an actual overwrite.